pkg/controllers/reallocation/utilization.go

```diff
@@ -122,6 +126,24 @@ func (u *Utilization) terminateExpired(ctx context.Context, provisioner *v1alpha
 	return nil
 }
 
+func (u *Utilization) terminateFailedToBecomeReady(ctx context.Context, provisioner *v1alpha3.Provisioner) error {
+	// 1. Get underutilized nodes
```

Comment:
This is all provisioner nodes. This comment and the error message on line 133 is inaccurate.